### PR TITLE
Minor Fixes to make master function correctly

### PR DESCRIPTION
**Pull request by Thunae**
Two Lines of Bug Fixs to make master function correctly

### CODE REVIEW

File: src/main/webapp/js/form.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
 /* ==========================================================================
    SEARCH FORM
    ========================================================================== */
-$("#randomize-form").submit((event) => {
+$("#randomize-form").submit(function(event) {
```
Comment (lyze) [inline src/main/webapp/js/form.js]: is this change necessary?

**Review by lyze — DISMISSED** [review]

File: src/main/webapp/js/form.js (hunk 1 above)
Comment (lyze) [reply src/main/webapp/js/form.js]: Oh I see. We would have to replace `$(this)` usages with `event.currentTarget` if we want to use arrows.

Comment (Thunae) [reply src/main/webapp/js/form.js]: Yea the arrow function isn't working with the AJAX selector and it crashes the app

Comment (Thunae) [reply src/main/webapp/js/form.js]: Should I use the arrow function and change the $this or keep the current state?

Comment (lcatherine) [reply src/main/webapp/js/form.js]: arrow function is generally preferred

**Review by lcatherine — APPROVED** [review]
Fix merge conflict
